src/PerformanceTable.h

```diff
@@ -142,7 +143,16 @@ class PerformanceTable {
    */
   std::vector<Perf> getWorstPerfByCrit(Criteria crits);
 
-private:
+  /**
+   * findAlt return true if the alternative is in the performance table
+   *
+   * @param altName Alternative to lookup
+   *
+   * @return true if found, false if nots
+   */
+  bool findAlt(std::string altName);
```

Comment:
nit: by convention we tend to use a question that can be answered by yes or no for the name of boolean functions: 

example: `isAltIn(std::string altName)`

it makes the code more readable when we read the code in conditions: 

```
if ( isAltIn("alt1") ) { }
```

Reply:
ok, will correct!